docs/how-to-write-a-contract/scriptcontext.md

```diff
@@ -140,13 +140,24 @@ But we can access the full prevouts via `this.prevouts`.
 
 [SigHash type](https://wiki.bitcoinsv.io/index.php/SIGHASH_flags) decides which part of the spending transaction is included in `ScriptContext`.
 ![](../../static/img/sighashtypes.png)
-It defaults to `SigHash.ALL`, including all inputs and outputs. You can customize it by setting the argument of the `@method()` decorator, e.g.,
+It defaults to `SigHash.ALL`, including all inputs and outputs. You can customize it by setting the argument of the `@method()` decorator, like:
 
 ```ts
+@method(SigHash.ANYONECANPAY_ALL)
```

Comment:
add a couple more examples to make the different SigHash functions more apparent